packages/experimental/src/global/js/package-settings.js

```diff
@@ -47,17 +47,56 @@ const defaults = {
   },
 };
 
-const setAll = (flags, enabled) => {
-  for (const key in flags) {
-    flags[key] = enabled;
-  }
-};
+const component = { ...defaults.component };
+const feature = { ...defaults.feature };
+const warningMessage = (type, property) =>
+  `IBM Cloud Cognitive (WARNING): ${type} "${property}" enabled via feature flags. This component has not yet completed it's review process.`;
+const warningMessageAll = (type) =>
+  `IBM Cloud Cognitive (WARNING): All canary ${type.toLowerCase()} have been enabled through use of setAll${type}`;
 
 const settings = {
+  _allComponents: false,
+  _allFeatures: false,
   // flags
   prefix: defaults.prefix,
-  component: { ...defaults.component },
-  feature: { ...defaults.feature },
+  component: new Proxy(component, {
+    set(target, property, value) {
+      console.warn(warningMessage('Component', property, value));
```

Comment:
I think the message only makes sense when a component is enabled, not if one gets disabled.

```suggestion
      if (value) {
        console.warn(warningMessageComponent(property));
      }
```